modules/vpc-peering/README.md

```diff
@@ -0,0 +1,211 @@
+# Component: `vpc-peering`
+
+This component is responsible for creating a peering connection between two VPCs existing in different AWS accounts.
+
+## Usage
+
+**Stack Level**: Regional
+
+Here's an example snippet for how to use this component.
+
+`stacks/vpc-peering/vpc-peering-defaults.yaml` (default VPC peering settings for all accounts):
+
+```yaml
+components:
+  terraform:
+    vpc-peering:
+      vars:
+        requester_allow_remote_vpc_dns_resolution: true
+        accepter_allow_remote_vpc_dns_resolution: true
+        accepter_region: <REGION>
+        accepter_vpc_id: <VPC ID>
+        accepter_aws_assume_role_arn: arn:aws:iam::<LEGACY ACCOUNT ID>:role/-vpc-peering
+```
+
+`stacks/ue1-sandbox.yaml`:
+
+```yaml
+import:
+  - vpc-peering/vpc-peering-defaults
+```
+
+
+## Legacy Account Configuration
```

Comment:
These docs were carried over from a customer and I've anonymized them. Worth considering if they'll be useful for others or not.